tests/explainers/test_tree.py

```diff
@@ -1641,6 +1642,64 @@ def test_lightgbm_call_explanation(self):
         assert (explanation.values == shap_values).all()
 
 
+class TestExplainerCausalML:
```

Comment:
we typically don't use classes in shap tests, would be great if you could refactor to use pytest.fixtures instead.

Also 
```python
        from causalml.dataset import synthetic_data
        from causalml.inference.tree import CausalTreeRegressor

        X, y, treatment = self.prepare_data(synthetic_data)
 ```       
        could be handled in those. Pls also consider the usage of `pytest.importorskip` so that tests pass even if causalml is not installed.

Reply:
Hi, @CloseChoice :wave: , can you review my latest changes, please?
Thanks